assets/training/aoai/proxy_components/src/finetune_submit.py

```diff
@@ -40,27 +46,72 @@ def submit_finetune_job(self, training_file_id, validation_file_id, model, regis
             validation_file=validation_file_id,
             hyperparameters=hyperparameters,
             suffix=suffix)
-        job_id = finetune_job.id
+        self.job_id = finetune_job.id
         status = finetune_job.status
```

Comment:
can we log status or finetune_job object itself?

Reply:
added log